### ISSUE TITLE
LibreOffice Portable 

### ISSUE THREAD

**Issue by ComradDart** [sbraconnier/jodconverter#254]
Hi, I've read wiki and [https://github.com/sbraconnier/jodconverter/issues/28](this), but still my program stucks with no trace. So I use windows xp and libreoffice portable 5.14. This is my code
![image](https://user-images.githubusercontent.com/8507587/112744322-7861fa80-8fe2-11eb-95e2-68a0a2d15db1.png)
officeHome variable defined as %libreofficeportabledir%\App\libreoffice
Whats wrong?
This is what I get
![image](https://user-images.githubusercontent.com/8507587/112744358-d1319300-8fe2-11eb-970a-0d5ba1b2f141.png)

P.S. It stucks at creating OfficeManager instanse

**Comment by sbraconnier** [comment]
Are you able to properly convert your document WITHOUT jodconverter ?? (juste by using libreoffice portable?).

@benelot Any ideas?

**Comment by benelot** [comment]
Wow, well that is too long ago. I do not remember, honestly.

**Comment by sbraconnier** [comment]
@benelot I was expecting that, but I never had to anwser a LibreOffice portable (Windows XP on top of that) question so I took a chance :) Thanks anyway!

**Comment by sbraconnier** [comment]
@ComradDart I don't see any jodconverter trace in your log. Could you please set the log level to TRACE and put back here the full log?

**Comment by benelot** [comment]
Ah, now that you told me I used the portable version, I found the old issue here. I think they contain all the important bits I had to do to make it work.
https://github.com/sbraconnier/jodconverter/issues/28#issuecomment-333811431
https://github.com/sbraconnier/jodconverter/issues/29#issuecomment-340710009
and following comments in those issues helped me. If any of the things were not as described, it did not work. Most importantly, an installed version of libreoffice on the host machine aside from the portable version.
